rule/struct_tag.go

```diff
@@ -562,13 +562,18 @@ func checkYAMLTag(checkCtx *checkContext, tag *structtag.Tag, _ ast.Expr) (messa
 
 func checkSpannerTag(checkCtx *checkContext, tag *structtag.Tag, _ ast.Expr) (message string, succeeded bool) {
 	if tag.Name == "-" {
+		if len(tag.Options) > 0 {
+			return fmt.Sprintf("useless option(s) %s for ignored field", strings.Join(tag.Options, ",")), false
+		}
 		return "", true
 	}
```

Comment:
Isn't the same for json and yaml tag then ? And maybe all of them ?

I mean when the value is - what could we expect from the serializer when it has option ?

Reply:
Yes, I will extend the check to those cases in a future PR